lnwallet/channel.go

```diff
@@ -4544,6 +4544,7 @@ func (lc *LightningChannel) computeView(view *HtlcView,
 	// need this to determine which HTLCs are dust, and also the final fee
 	// rate.
 	view.FeePerKw = commitChain.tip().feePerKw
+	view.NextHeight = nextHeight
```

Comment:
Set, but not added in this commit?

Reply:
Yeah, we added it earlier in part 2. But looks like I forgot to squash this commit with an earlier one, so this remained.